iquip/apps/logger.py

```diff
@@ -172,21 +174,21 @@ def frames(self) -> Tuple[LoggerFrame]:
 
     @pyqtSlot(str)
     def addLog(self, content: str):
-        """Adds a channel name and log message.
+        """Adds a received log message to the LoggerFrame.
 
         Args:
-            content: Received log message.
+            content: A received log message.
```

Comment:
You don't need "A" here since it is just a noun and describes the meaning of `content`.
If you want to use an article, I think "the" is more appropriate because it refers to the specific log message that is passed as the `content` parameter.

Reply:
I think these are quite subtle and you only have to be careful when you write sentences.

Reply:
I will go back to previous one.